### PR TITLE
Configure display setting of Dataframe in 2.0 tutorial

### PR DESCRIPTION
**Pull request by JohnsonKuan**
Configure display setting of Dataframe in 2.0 tutorial to address issues with background color of header in dark mode for docs.

Background color of rows will be addressed in another PR after 2.0 tagging.

```
from IPython.core.display import HTML
HTML("""
<style>
    .dataframe {
        background: #D7D7D7;
    }
    
    th {
        color:black;
    }
</style>
""")
```

Preview in dark mode of Jupyter Lab:
image

Preview in non-dark mode:
image

### CODE REVIEW

**Comment by jwmueller** [comment]
Won't this be visible in the docs build and in the colab notebook?

File: docs/source/tutorials/indepth_overview.ipynb [hunk 1]
```diff
@@ -77,6 +77,24 @@
     "%config InlineBackend.print_figure_kwargs={\"facecolor\": \"w\"}"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%HTML\n",
```
Comment (jwmueller) [inline docs/source/tutorials/indepth_overview.ipynb:86]: Won't this be visible in the docs build and in the colab notebook?

**Review by cgnorthcutt — APPROVED** [review]
LGTM. Double check jonas's comment. After merge, double check it worked (takes a few minutes to load) and double check these settings and weird code blocks are not visible to user.

File: docs/source/tutorials/indepth_overview.ipynb (hunk 1 above)
Comment (jwmueller) [reply docs/source/tutorials/indepth_overview.ipynb:86]: At least need to replace `metadata` above with:

```"metadata": {
    "nbsphinx": "hidden"
   },
```

That will at least hide in the docs website.

Comment (JohnsonKuan) [reply docs/source/tutorials/indepth_overview.ipynb:86]: Yes I believe so, the code will be visible. I'm not sure how to hide it though.

Comment (jwmueller) [reply docs/source/tutorials/indepth_overview.ipynb:86]: Actually if you want to add this stuff to this notebook it should go into the cell that is already hidden (the one starting with `# Package installation.`)

Comment (JohnsonKuan) [reply docs/source/tutorials/indepth_overview.ipynb]: Sounds good, agree. I moved it to the cell that is already hidden.

**Pull request merged**